docs/index.html

```diff
@@ -1,11 +1,106 @@
-<!DOCTYPE html>
 <html>
-<head>
-	<meta http-equiv="refresh" content="0; url=purchases/index.html" />
-</head>
-<body>
```

Comment:
not sure this should have been versioned in the first place, but not looking to change that right now

Reply:
ohhh interesting, makes sense. I'll kill it then

Reply:
doen